widget_driver/test/widget_driver_test.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:widget_driver/widget_driver.dart';
+
+// ignore: avoid_relative_lib_imports
+import '../../widget_driver_test/lib/widget_driver_test.dart';
```

Comment:
Now here I am importing a file by path from the `widget_driver_test` package.. This you usually should not do.

But here it is safe to do it since this is a test.. And tests are not deployed.. So this code will always only run when it as access to the direct path to the `widget_driver_test` package..

Reply:
I don't want to add a dependency in the pubspec.yaml to the `widget_driver_test`... Since this is the `widget_driver`. And the `widget_driver_test` has a dependency to `widget_driver`.

So then I cannot have the `widget_driver` also depend on the `widget_driver_test`..